Nasal/FMGC/FMGC.nas

```diff
@@ -625,7 +628,7 @@ var masterFMGC = maketimer(0.2, func {
 	
 	if (FMGCInternal.phase == 1 and ((mode != "SRS" and mode != " ") or alt >= accel_agl_ft)) {
 		FMGCInternal.phase = 2;
-		systems.PNEU.pressMode.setValue("TO");
+		systems.PNEU.pressMode.setValue("TO"); # TODO - why not set to CL (climb) ??
```

Comment:
Please ignore pressurization for now. That's rather broken and needs re-done.

Reply:
yep. 
I've made a typo for computed_mach.
I found thales manual (rev1, I think is a lot old), I'm reading for check differences.